project/Settings.scala

```diff
@@ -142,6 +142,9 @@ object Settings {
     val guava                   = "25.0-jre"
     val prometheusClient        = "0.6.0"
     val geminiLocales           = "0.1.0-2019a"
+
+    // Gemini Libraries
+    val gspMath                 = "0.1.0+5-7322fff9-SNAPSHOT"
```

Comment:
Let's tag a real version here. `0.1.1` maybe.

Reply:
Is the version with the hash published after you merge a PR right? If so the hash would never change if master is protected

Reply:
Yes, that's the merge commit. So it's really not a snapshot since it's stable, but I think we probably don't want every downstream project to get an update PR every time there's a merge into master. So, work for a while, use the snapshot in dev if you need to, then tag a release when it seems time.

Reply:
We make updates to libraries like `gsp-math` *because we need them* for one or more of our applications. I don't think we're likely to work for a while on a library independently and then tag it.  Instead it seems more likely that we'll be trying to update an application, find that we need some new syntax or a new method and have to switch to the library to add it.  If we need to tag every update before using it we'll be tagging with every tweak.  I suppose that's fine and I will do it if that's the consensus, but I guess I don't see it as really necessary at least while everything is under active development.

Reply:
We can probably live with just the hash and only tag every now and then when appropriate.

I'd personally not spawn a library out until it is somewhat stable. It is going to be annoying to be switching projects during active development

Reply:
I think shared libraries like `gsp-math` and whatever we call the library that contains model code are going to be under constant development.  It will be quite a while before everything stabilizes.